config/locales/forms.yml

```diff
@@ -227,6 +243,29 @@ en:
       publishers_job_listing_pay_package_form:
         benefits: Employee benefits
         salary_html: Salary (<span class='text-red'>Required</span>)
+      personal_details_form:
+        building_and_street: Building and street
+        email_address_html: Email address (<span class='text-red'>Required</span>)
+        first_name_html: First name (<span class='text-red'>Required</span>)
+        last_name_html: Last name (<span class='text-red'>Required</span>)
+        national_insurance_number_html: National insurance number (<span class='text-red'>Required</span>)
```

Comment:
National Insurance

Reply:
The i is lower case in the designs!

Reply:
Content in the designs can contain mistakes.

Reply:
Acutally, I've just realised it's uppercase in the label but upper cased when the number is invalid. Will talk to content now to see which they prefer.

Reply:
Sorted. National Insurance it is.